vernac/egramcoq.ml

```diff
@@ -111,7 +111,7 @@ let find_position_gen current ensure assoc lev =
         else if admissible_assoc (a,assoc) then
           raise_notrace Exit
         else
-          error_level_assoc p a (Option.get assoc)
+          raise @@ NotationLevelMismatch (p, a, Option.get assoc)
```

Comment:
This is great, you can keep the `Error.level_assoc` function tho as a wrapper over the raise + constructor, this is common practice in industrail OCaml programming as to hide the concrete implementation of the actual exn.

Reply:
Alright, I'll keep the `error_level_assoc` as a wrapper then.